apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/MultiHostValidatorOperatorIntegrationTest.scala

```diff
@@ -0,0 +1,199 @@
+        // set the ContractImportMode, so we use the repair.import_acs command instead.
+        bobParticipant.repair.import_acs(
+          importFilePath = acsFile.toString
+        )
+      }
+    })
+
+    clue("Reconnect synchronizer and start the validator app again") {
+      bobParticipant.synchronizers.reconnect_all()
+      bobValidatorBackend.startSync()
+    }
+
+    actAndCheck(
+      "Clear the onboarding flags", {
+        multiHostedParties.map(party =>
+          bobParticipant.topology.party_to_participant_mappings
+            .propose_delta(
+              party,
+              adds = Seq((bobParticipantId, ParticipantPermission.Observation)),
+              store = synchronizerId,
+            )
+            .transaction
+            .operation
+        )
+      },
+    )(
+      "Alice sees the transaction",
+      // FIXME: in the docs, this is "wait for decision timeout". Can we safely replace it with "alice sees the topology tx"?
+      _.foreach(op =>
+        aliceParticipant.topology.party_to_participant_mappings
+          .list(synchronizerId, operation = Some(op)) should not be empty
+      ),
+    )
+
+    multiHostedParties.foreach(party =>
+      clue(s"Set permission to confirmation for $party") {
+        Seq(aliceValidatorBackend, bobValidatorBackend).map(
+          _.participantClient.topology.party_to_participant_mappings
+            .propose_delta(
+              party,
+              adds = Seq((bobParticipantId, ParticipantPermission.Confirmation)),
+              store = synchronizerId,
+            )
+            .transaction
+            .operation
+        )
+      }
+    )
+
+    clue("Setup transfer preapproval") {
+      // Tap some amulet, so the validator has funds to cover the transfer preapproval creation fee
+      aliceValidatorWalletClient.tap(10)
+      createTransferPreapprovalEnsuringItExists(aliceWalletClient, aliceValidatorBackend)
+    }
+
+    aliceWalletClient.balance().unlockedQty should beAround(0.0)
+    actAndCheck(
+      "Stop alice validator app, and disconnect its participant", {
+        aliceValidatorBackend.stop()
+        aliceParticipant.synchronizers.disconnect_all()
+      },
+    )(
+      "The participant is disconnected",
+      _ => aliceParticipant.synchronizers.list_connected() shouldBe empty,
+    )
+
+    splitwellWalletClient.tap(walletAmuletToUsd(100))
+    splitwellWalletClient.balance().unlockedQty should beAround(100.0)
+    val deduplicationId = UUID.randomUUID.toString
+    val transferDescription = Some("test-description")
+    actAndCheck(
+      "Splitwell sends Alice 40.0 amulet",
+      splitwellWalletClient.transferPreapprovalSend(
+        aliceUserParty,
+        40.0,
+        deduplicationId,
+        transferDescription,
+      ),
+    )(
+      "The send succeeds despite alice's validator being disconnected and stopped",
+      _ => {
+        // Fees eat up quite a bit
```

Comment:
Use the zero fees config: https://github.com/hyperledger-labs/splice/blob/1084033944f1345151324aa5e60ee7fa43d25987/apps/app/src/main/scala/org/lfdecentralizedtrust/splice/config/ConfigTransforms.scala#L299